idemp-rest/WEB-INF/src/org/bandahealth/idempiere/rest/filter/AuthenticationFilter.java

```diff
@@ -60,16 +60,16 @@ public void filter(ContainerRequestContext requestContext) throws IOException {
 				if (Util.isEmpty(Env.getContext(Env.getCtx(), Env.AD_USER_ID))
 						|| Util.isEmpty(Env.getContext(Env.getCtx(), Env.AD_ROLE_ID))) {
 					if (!requestContext.getUriInfo().getPath().startsWith(IRestConfigs.AUTHENTICATION)) {
-						requestContext.abortWith(Response.status(Response.Status.UNAUTHORIZED).build());
+						requestContext.abortWith(Response.status(Response.Status.FORBIDDEN).build());
```

Comment:
Any thoughts on using 404s here instead of 403s? That will make our endpoints a bit securer.